docs/modules/devices/pages/local-devices/bypass-double-click-payment-confirmation-ios/setup-process.adoc

```diff
@@ -0,0 +1,138 @@
+= Setup Process
+:navtitle: Setup Process
+
+You can perform the setup process physically on the device or through a Kobiton Manual Session:
+
+. xref:_on_the_device[]
+. xref:_in_a_manual_session[]
+
+== Requirements
+
+- Access to the physical device to double-click the side button during setup of AssistiveTouch. (If the device is hosted by Kobiton, submit a support ticket to assist in the setup process.)
+- A passcode set on the device. There are two options:
+* A passcode can be set *and must be removed* during the manual session. If the passcode is not removed before ending the session, the device *will not be able to come back online* without physical touch and a request to the team managing the devices to unlock the device.
+* Setup Kobiton passcode support for the devices. For Private devices, discuss passcode requirements with your Customer Success Manager who will work with support to setup passcodes. For Local devices hosted at the customer location, submit a support ticket to request passcode setup documentation for your deployment type (Legacy Kobiton or PIN code mode 1 for Kobiton 4+)
+- [Optional] Use a link:https://support.kobiton.com/hc/en-us/articles/360055588272-Device-Cleanup[Cleanup Policy] for the devices to be tested that does not reset the device settings to prevent AssistiveTouch from turning off for the next session. Otherwise, the tester will need to turn on AssistiveTouch for each session that it is needed:
+
+image::devices:bypass-apple-pay-configure-cleanup-popup.PNG[]
+
+
+== On the Device
+
+After the device has been set up with passcode management, start a Manual Session on the device. The system generates a random passcode and sets it on the device in the background.
+
+*Step 1:* Navigate to *Settings → Face ID & Passcode*. If you see the below screen asking for a passcode, then passcode is enabled on the device.
```

Comment:
@Hikkigaya can you remove Step + number here as well? Also remove them for the other document.

Reply:
@Hikkigaya never mind. I'll work on this

Reply:
Added a commit to remove steps.